### PR TITLE
New bitcoin dev scraper

### PR DESCRIPTION
**Pull request by urvishp80**
- bitcoin-dev for https://gnusha.org/pi/bitcoindev/
- github workflow

### CODE REVIEW

**Comment by adamjonas** [comment]
Will this scrape all the posts from the Linux mailing list as well?

**Comment by urvishp80** [comment]
> Will this scrape all the posts from the Linux mailing list as well?

No, because Linux foundation (bitcoin-dev) stopped updating new threads after 9th Feb 2024. All of its threads past threads have been migrated to gnusha.org. 
So, 
1. The new threads will be posted here (gnusha.org) only and not on Linux foundation mailing list. It will scrap them.
2. If there are new replies to a main thread belonging to Linux foundation it will scrape the new replies only and insert their domain, URL etc. accordingly.   

Basically, our database will have all the threads from Linux foundation mailing list and gnusha.org without being duplicated.

**Comment by adamjonas** [comment]
I understand it's not posting in both places but will this scraper pick up the already posted Linux threads on gnusha?

**Comment by urvishp80** [comment]
Yes, It will scraper the already posted linux threads, but it will not insert them into the ES index. Because we have added a check to make sure the same data does not get re-inserted again.

**Comment by adamjonas** [comment]
I merged it but I guess I don't feel 100% on the reasoning that the already posted linux threads won't be added to ES given this is a different domain with possibly different formatting. Looking forward to be proven wrong!
